src/Moryx/Serialization/DefaultSerialization.cs

```diff
@@ -103,30 +111,38 @@ public virtual EntryValidation CreateValidation(Type memberType, ICustomAttribut
             // Iterate over attributes reading all validation rules
             foreach (var attribute in validationAttributes)
             {
-                if (attribute is MinLengthAttribute minAttribute)
-                {
-                    validation.Minimum = minAttribute.Length;
-                }
-                else if (attribute is MaxLengthAttribute maxAttribute)
-                {
-                    validation.Maximum = maxAttribute.Length;
-                }
-                else if (attribute is RangeAttribute rangeAttribute)
+                switch (attribute)
                 {
-                    validation.Minimum = Convert.ToDouble(rangeAttribute.Minimum);
-                    validation.Maximum = Convert.ToDouble(rangeAttribute.Maximum);
-                }
-                else if (attribute is RegularExpressionAttribute regexAttribute)
-                    validation.Regex = regexAttribute.Pattern;
-                else if (attribute is StringLengthAttribute strLength)
```

Comment:
Am I missing something or are the `RegularExpressionAttribute` and the `StringLengthAttribute` missing? 🤔